src/derivative_utils.jl

```diff
@@ -1,6 +1,6 @@
 const ROSENBROCK_INV_CUTOFF = 7 # https://github.com/SciML/OrdinaryDiffEq.jl/pull/1539
 
-struct StaticWOperator{isinv, T}
+struct StaticWOperator{isinv, T} <: AbstractSciMLOperator{T}
```

Comment:
what is the effect here? Can you time a static array ROBER and make sure that doesn't regress?